packages/dev/test/provider/AccountManager.test.ts

```diff
@@ -21,7 +21,7 @@ contract('AccountManager', function (accounts) {
   const address = '0x982a8CbE734cb8c29A6a7E02a3B0e4512148F6F9'
   const privateKey = '0xd353907ab062133759f149a3afcb951f0f746a65a60f351ba05a3ebf26b67f5c'
   const config = configureGSN({
-    methodSuffix: '',
+    methodSuffix: '_v4',
```

Comment:
the default should work, shouldn't it?

Reply:
Another difference between ganahce and hardhat, method suffix needs to be "_v4" in hardhat node.

Reply:
yeah, the 'getDefaultMethodSuffix' should work, right?